packages/realm-server/server.ts

```diff
@@ -656,6 +658,13 @@ export class RealmServer {
     throw new Error(`Can not determine the matrix registration secret`);
   }
 
+  private handleStripeWebhook = async (ctxt: Koa.Context, _next: Koa.Next) => {
+    let request = await fetchRequestFromContext(ctxt);
+
+    let response = await stripeWebhookHandler(this.dbAdapter, request);
```

Comment:
Just a note that we will want this to be handled in an async job that gets queued in a high priority queue (so that we can have quick webhook acknowledgement response times to stripe and retry mechanisms).

Until we have that I think it's ok we `await` the handler. But in case an error happens in the handler I suggest sending the whole webhook data object to Sentry so we'll know exactly what data we received if something goes wrong.